modules/schema/src/main/java/org/apache/ignite/internal/schema/Column.java

```diff
@@ -30,6 +30,9 @@
  * user-defined order ({@link #columnOrder}).
  */
 public class Column implements Serializable {
+    /** Default "default value supplier". */
+    private static final Supplier<Object> NULL_SUPPLIER = () -> null;
```

Comment:
NULL_SUPPLIER is not serializable now.
Seems, we can remove Serializable interface from this class,  SchemaDescriptor, and others as IGNITE-14679 already merged.

Reply:
Remove Serializable from SchemaDescriptor, Columns, Column, NativeType, ColumnMapper, ColumnMapping.